quickstart/prerequisites.md

```diff
@@ -5,37 +5,208 @@
 All worker nodes must satisfy the following requirements:
 
 * **x86-64** CPU cores with SSE4.2 instruction support:
-  * Intel Nehalem processor \(march=nehalem\) and newer
+  * First generation Intel Core i5 and Core i7 (Nehalem microarchitecture) or newer
   * AMD Bulldozer processor and newer
 
 * Linux kernel **5.4 or higher** with the following modules loaded:
   * nvme-tcp
   * ext4 and optionally xfs
 
-## **Mayastor pods (Data Plane/Engine)**
-Each worker node which will host an instance of a Mayastor pod must have the following resources _free and available_ for use by Mayastor:
+* [Helm](https://helm.sh/docs/intro/install/) version must be v3.7 or later.
 
-* Two CPU cores
-* 4GiB RAM
-* **HugePage support**
-  * A minimum of **2GiB of** **2MiB-sized** pages
+* Supported Kubernetes versions: 
```

Comment:
These aren't _supported_ versions.  These are the versions we test on.  There's a difference.  Also, I'm not sure why this information is being placed under the prerequisites banner.  We don't mandate the use of these versions.

Reply:
@avishnu Even if they are "recommended", that's not the same as being a prerequisite.

Reply:
Removing this section from prerequisite page.